spyder/widgets/findinfiles.py

```diff
@@ -139,8 +139,10 @@ def find_files_in_path(self, path):
                         if self.stopped:
                             return False
                     dirname = os.path.join(path, d)
-                    if self.exclude \
-                       and re.search(self.exclude, dirname + os.sep):
+                    if (self.exclude and

```

Comment:
Break before binary operator?

Reply:
I don't understand what you mean here.

Reply:
https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

It seems our code generally follows the modern "break before the operator", including that bit before you modified it.

Reply:
PEP 8 recommends it, but doesn't require it. Just wanted to put that out there since that seems to be the convention a lot of our code follows, including the modified portion.

Also, just FYI, PEP8Speaks doesn't seem to be really actively maintained right now; the last commit is almost five months ago and my issues and PRs have been ignored unanswered for over three. However, the maintainer may simply be on hiatus since I did see him post one issue himself a week ago, so there's hope.